datafusion/physical-plan/src/aggregates/mod.rs

```diff
@@ -2489,4 +2497,49 @@ mod tests {
 
         Ok(())
     }
+
+    #[test]
+    fn group_exprs_nullable() -> Result<()> {
```

Comment:
I think it is possible to add the test in slt 🤔 ? Only create rust test if it is not trivial to add test in slt

Reply:
Yeah, not know how to easily test schema in slt.